src/backend/marsha/core/models/file.py

```diff
@@ -136,6 +136,7 @@ class Document(BaseFile):
     """Model representing a document."""
 
     RESOURCE_NAME = "documents"
+    FILE_TYPE_NAME = _("document")
```

Comment:
This can now be removed.

Reply:
Why ? I use it line 152 and 153

Reply:
Oh yes the entire string can be translated... sorry